crates/ruff_linter/src/rules/flake8_use_pathlib/violations.rs

```diff
@@ -1215,3 +1215,45 @@ impl Violation for OsListdir {
         "Use `pathlib.Path.iterdir()` instead.".to_string()
     }
 }
+
+/// ## What it does
+/// Checks for uses of `os.symlink`.
+///
+/// ## Why is this bad?
+/// `pathlib` offers a high-level API for path manipulation, as compared to
+/// the lower-level API offered by `os.symlink`.
+///
+/// ## Example
+/// ```python
+/// import os
+///
+/// os.symlink("usr/bin/python", "tmp/python", target_is_directory=False)
+/// ```
+///
+/// Use instead:
+/// ```python
+/// from pathlib import Path
+///
+/// Path("usr/bin/python").symlink_to("tmp/python")
```

Comment:
I think it should be the opposite. But please verify it locally
```suggestion
/// Path("tmp/python").symlink_to("usr/bin/python")
```